website/docs/guides/version_4_upgrade.html.markdown

```diff
@@ -0,0 +1,196 @@
+---
+layout: "vault"
+page_title: "Terraform Vault Provider 4.0.0 Upgrade Guide"
+sidebar_current: "docs-vault-provider-version-4-upgrade"
+description: |-
+  Terraform Vault Provider 4.0.0 Upgrade Guide
+
+---
+
+# Terraform Vault Provider 4.0.0 Upgrade Guide
+
+Version `4.0.0` of the Vault provider for Terraform is a major release and
+includes some changes that you will need to consider when upgrading. This guide
+is intended to help with that process and focuses only on the changes necessary
+to upgrade from version `3.24.0` to `4.0.0`.
```

Comment:
Should this be "3.24.0 or above"? Same with the other references that mentions 3.24.0 directly.

Reply:
Yes, I can do that. 3.24 was a placeholder as I wasn't sure what version we would be on at the time of release.